demo/src/app/components/datepicker/demos/adapter/datepicker-adapter.html

```diff
@@ -1,6 +1,10 @@
-<p>These datepickers use custom Date adapter that lets you use your own model implementation.
-In this example we are converting from and to a JS native Date object</p>
-
+<p>These datepickers use custom date adapter and formatter.
+The custom NgbDateAdapter lets you use your own model i.e. the structure of ngModel.
```

Comment:
wrap with `<code>NgbDateAdapter</code>`, so it renders with visual clue <code>NgbDateAdapter</code>